testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/util/IcebergTestUtils.java

```diff
@@ -18,14 +18,18 @@
 
 import static com.google.common.base.Verify.verify;
 import static io.trino.tests.product.utils.QueryExecutors.onTrino;
+import static java.lang.String.format;
 
 public final class IcebergTestUtils
 {
+    private static final String HDFS_URI = "hdfs://hadoop-master:9000";
+
     private IcebergTestUtils() {}
 
-    public static String getTableLocation(String tableName)
+    public static String getTableLocation(String tableName, boolean withURI)
```

Comment:
by looking at  the methoid signature, I don't understand  what "withURI" means. 
Why do we want a change here?

Reply:
Some test cases were already using `getTableLocation` where the caller gets the `tableLocation` without`hdfs://hadoop-master:9000` (hdfsURI). For me, I wanted to have a fully qualified `tableLocation` to call `registerTable`.

Here `withURI` means, return the table location which includes `hdfs://hadoop-master:9000` 

That is why I have overloaded the method to not disturb the existing caller.

Reply:
For now, I have reverted my change and returned the table location with `hdfs://hadoop-master:9000` always. My mac has issue with product tests, so relying on CI build to see the error if any.

`

Reply:
The test cases are failing.
It is found that `hdfsClient` requires table location **without** `hdfs://hadoop-master:9000` and `register_table` procedure call requires table location **with** `hdfs://hadoop-master:9000`

Reply:
is `withNamenodeURI` make sense?